noxfile.py

```diff
@@ -72,7 +72,13 @@
 UNIT_TEST_LOCAL_DEPENDENCIES: List[str] = []
 UNIT_TEST_DEPENDENCIES: List[str] = []
 UNIT_TEST_EXTRAS: List[str] = []
-UNIT_TEST_EXTRAS_BY_PYTHON: Dict[str, List[str]] = {"3.12": ["polars"]}
+UNIT_TEST_EXTRAS_BY_PYTHON: Dict[str, List[str]] = {
+    "3.9": ["scikit-learn"],
+    "3.10": ["scikit-learn"],
+    "3.11": ["scikit-learn"],
+    "3.12": ["polars", "scikit-learn"],
+    "3.13": ["scikit-learn"],
```

Comment:
No.

We can't install it in _every_ session. We need to know that bigframes works without this package!

```suggestion
    "3.12": ["polars", "scikit-learn"],
```

Reply:
Updated, but I assume in this case unit noextras sessions runs the same thing as it test?